nsqd/http.go

```diff
@@ -226,6 +231,32 @@ func (s *httpServer) getTopicFromQuery(req *http.Request) (url.Values, *Topic, e
 	return reqParams, s.ctx.nsqd.GetTopic(topicName), nil
 }
 
+func (s *httpServer) doLookup(req *http.Request) (interface{}, error) {
```

Comment:
Yes, I just want it to be easy and obvious to both programmatically or manually inspect/confirm the state of a node WRT to the gossip cluster (this seems like an obvious thing to alert on).